packages/dcos-ui/buildinfo.json

```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-ui.git",
-    "ref": "89b9fbd36582d107e67bea114151c4c5504b93a3",
-    "ref_origin": "release/1.9"
+    "ref": "62d63a79793e428596c7c631b12db2d7d68ab170",
+    "ref_origin": "integrate/marathon-1.5"
```

Comment:
Thanks @orlandohohmeier ; Anything that will help this PR get landed faster I think will make everything easier.